docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.9"
 
 services:
-  api:
+  api: &api
```

Comment:
I've never seen this syntax - what does it do?

Reply:
It creates a shortcut so that the `seed` script references the `api` container